frontend/src/js/components/releases/DeltaGenerationDetailsDrawer.tsx

```diff
@@ -299,7 +257,7 @@ export const DeltaGenerationDetailsDrawer = ({ jobId, onClose, open }: DeltaGene
             </div>
             <LinedHeader className="margin-top-large" heading="Status" />
             <DetailsTable className={classes.table} columns={statusColumns} items={[combinedData]} />
-            {combinedData.status === 'failed' && combinedData.log && <Code className="log">{combinedData.log}</Code>}
```

Comment:
Interesting. AFAIK there never was any serialized attribute with the name `log`, it's always been `details` but there is an OAS example with an attribute log :thinking: